hack/golangci.yaml

```diff
@@ -14,155 +14,109 @@
 run:
   timeout: 30m
 
+  # The default is relative to the configuration, which is confusing because
+  # then all paths start with ../ to move out of the "hack" directory.
+  # `gomod` mirrors the current behavior of `golangci-lint.sh` changing into
```

Comment:
This makes sense to me, this is essentially the repo root, at least for the main module.

That will make other paths more predictable anyhow.